docs(k8s): add faq entry (#1501)
scaleway/docs-content: faq/kubernetes.mdx

If the workload of your infrastructure is low, only the minimum number of machines will be running. If the load grows, additional machines are started automatically up to the maximum number of machines in the pool, to handle the workload on your application. If the load decreases, these additional machines will be stopped automatically.

## Can I change the type of my cluster after creation (Kapsule to Kosmos, shared to dedicated, etc.)?

The `SetClusterType` API call allows you to upgrade your cluster's control plane from shared to dedicated plans (currently available with 8 GB and 16 GB control planes). However, please keep in mind that certain conversions are not possible.

> For example, if you want to move your workloads from Kapsule to Kosmos or from Kosmos to Kapsule, or if you want to downgrade from a dedicated control plane to a shared plan, you'll need to create a new cluster.

Reviewer: ```suggestion
For example, moving your workloads from Kapsule to Kosmos or from Kosmos to Kapsule, or downgrading from a dedicated control plane to a shared plan, is not possible. You will need to create a new cluster.
```

Please refer to the [API documentation](https://developers.scaleway.com/en/products/k8s/api/#post-498231) for more information.

## How can I have data persistence on my cluster?

Kapsule is a managed Kubernetes engine. By definition, the nodes of your clusters can be deleted, replaced, or restarted if the applications running on it require it or if a node suddenly stops responding. It means that Kubernetes clusters' nodes are to be considered stateless.
If you require a stateful application, you can use [Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
The storageClass for Scaleway Block Storage volumes is set by default, so it does not need to be specified.
For more information, refer to the Kubernetes Kapsule product documentation on [GitHub](https://github.com/scaleway/scaleway-csi/tree/master/examples/kubernetes).

## What can I do if my system volume is running out of space?

> To ensure sufficient space for storing default system files and configurations, it is recommended that the capacity of your system volume disk should be at least 20GB. While this amount is generally sufficient for small workloads, it is advisable to have a capacity of 100GB for comfortably storing images and system logs in many cases.

Reviewer: ```suggestion
To ensure sufficient space for storing default system files and configurations, we recommend a system volume disk capacity of at least 20GB. While this amount is generally sufficient for small workloads, it is advisable to have a capacity of 100GB to comfortably store images and system logs in many cases.
```

> If you run out of space on your immutable system volume disk, you can create another pool and utilize the **Advanced Options** button to set your desired system volume type and size.

Reviewer: ```suggestion
If you run out of space on your immutable system volume disk, you can create another pool and use the **Advanced Options** button to set your desired system volume type and size.
```
